e2e/pages/backendUrl.page.js

```diff
@@ -0,0 +1,24 @@
+const _ = require('lodash');
+
+function fragments(text) {
+  const mapping = {
+    urlInput: element.bind(null, by.css('app-backend-url input')),
+    saveButton: element.bind(null, by.css('app-backend-url .save-button'))
+  };
+  return _.get(mapping, text);
+}
+
+function enterMockBackendUrl() {
+  return fragments('urlInput')().sendKeys('http://localhost:8060');
+}
+
+function confirm() {
+  return fragments('saveButton')().click();
```

Comment:
We don't have to use `fragments` and string path. Simply reference `mapping.saveButton().sendKeys(...)`